IaaS-Story/11-MultiNIC/azuredeploy.json

```diff
@@ -0,0 +1,289 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vnetName": {
+      "type": "string",
+      "defaultValue": "WTestVNet",
+      "metadata": {
+        "description": "Name for the new VNet."
+      }
+    },
+    "backEndSubnetName": {
+      "type": "string",
+      "defaultValue": "BackEnd",
+      "metadata": {
+        "description": "Name for the back end subnet."
+      }
+    },
+    "backEndNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-BackEnd",
+      "metadata": {
+        "description": "Name for the NSG used to allow DB access, and block Internet."
+      }
+    },
+    "remoteAccessNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-RemoteAccess",
+      "metadata": {
+        "description": "Name for the NSG used to allow remote access."
+      }
+    },
+    "stdStorageName": {
```

Comment:
For now I am leaving those there. And the documentation states they need to change the name to a unique name. I also do NOT have a default value. The reason I am avoiding uniqueString is that enterprise customers have a naming convention for their resources. And I am putting together someguidelines for that naking convention.

From: Kay Singh [mailto:notifications@github.com]
Sent: Wednesday, November 11, 2015 9:54 AM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Telmo Sampaio <telmos@microsoft.com>
Subject: Re: [azure-quickstart-templates] Added multiNIC documentation sample (#768)


In IaaS-Story/11-MultiNIC/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/768#discussion_r44562692>:

> +    },

> +    "backEndNSGName": {

> +      "type": "string",

> +      "defaultValue": "NSG-BackEnd",

> +      "metadata": {

> +        "description": "Name for the NSG used to allow DB access, and block Internet."

> +      }

> +    },

> +    "remoteAccessNSGName": {

> +      "type": "string",

> +      "defaultValue": "NSG-RemoteAccess",

> +      "metadata": {

> +        "description": "Name for the NSG used to allow remote access."

> +      }

> +    },

> +    "stdStorageName": {

Consider auto-generating this using uniqueString() function to prevent a bunch of failures

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/768/files#r44562692>.